packages/calcite-components/src/components/tree-item/tree-item.tsx

```diff
@@ -201,7 +201,10 @@ export class TreeItem implements ConditionalSlotComponent, InteractiveComponent
 
   render(): VNode {
     const rtl = getElementDir(this.el) === "rtl";
-    const showBulletPoint = this.selectionMode === "single" || this.selectionMode === "children";
```

Comment:
Do existing screenshot tests show changes? If not, can we add a screenshot test for he bullet point?